src/test/ui/generics/wrong-number-of-args.stderr

```diff
@@ -149,30 +145,30 @@ note: struct defined here, with 2 type parameters: `A`, `B`
 LL |     struct Ty<A, B>;
    |            ^^ -  -
 
-error[E0107]: missing generics for struct `lifetime_and_type::Ty`
-  --> $DIR/wrong-number-of-args.rs:44:14
+error[E0107]: this struct takes 1 type argument but 0 type arguments were supplied
+  --> $DIR/wrong-number-of-args.rs:43:14
    |
 LL |     type A = Ty;
    |              ^^ expected 1 type argument
    |
 note: struct defined here, with 1 type parameter: `T`
-  --> $DIR/wrong-number-of-args.rs:42:12
+  --> $DIR/wrong-number-of-args.rs:41:12
    |
 LL |     struct Ty<'a, T>;
    |            ^^     -
-help: use angle brackets to add missing type argument
+help: add missing type argument
    |
-LL |     type A = Ty<T>;
+LL |     type A = Ty, T;
```

Comment:
This is bad. It currently results from implicit lifetime arguments. We currently use the presence of generic arguments  to append missing arguments, but with implicit generic arguments there are no angle brackets and previous args to concatenate the missing arguments to. If this is a dealbreaker, I'll try to change this.

Reply:
We would ideally want to tackle this in this PR.

Reply:
fixed.